ceph-salt-formula/salt/ceph-salt/apply/sysctl.sls

```diff
@@ -0,0 +1,28 @@
+{% import 'macros.yml' as macros %}
+
+{% if 'cephadm' in grains['ceph-salt']['roles'] %}
+
+{{ macros.begin_stage('Configure sysctl') }}
+
+/etc/sysctl.d/90-ceph.conf:
```

Comment:
Oops. `/etc/sysctl.d` is reserved for local administrators, so they can override vendor settings like the ones we are creating in this PR.

I also think "ceph-salt" is a better label than "ceph", since the name `ceph-salt` will more effectively signal which tool is responsible for creating the file.

So, could you please change this to `/usr/lib/sysctl.d/90-ceph-salt.conf`?

Reply:
eh debatable, since since salt is a configuration management/remote administration tool and not a package manager (RPM).